tests/networkpolicy_test.go

```diff
@@ -71,6 +71,20 @@ var _ = Describe("[rfe_id:150][crit:high][vendor:cnv-qe@redhat.com][level:compon
 		vmic, err = virtClient.VirtualMachineInstance(tests.NamespaceTestAlternative).Get(vmic.Name, &v13.GetOptions{})
 		Expect(err).ToNot(HaveOccurred())
 
+		By("Checking that the VirtualMachineInstance vmia console has expected output, and configurating ipv6")
+		expecter, err := tests.LoggedInCirrosExpecter(vmia)
+		Expect(err).ToNot(HaveOccurred())
+		expecter.Close()
+
+		By("Checking that the VirtualMachineInstance vmib console has expected output, and configurating ipv6")
+		expecter, err = tests.LoggedInCirrosExpecter(vmib)
+		Expect(err).ToNot(HaveOccurred())
+		expecter.Close()
+
+		By("Checking that the VirtualMachineInstance vmic console has expected output, and configurating ipv6")
```

Comment:
Instead of `WaitForSuccessfulVMIStart` you can use `WaitUntilVMIReady` this takes a expector factory function so it exactly what we need here.

Reply:
yep thx this what i meant with "or other shortcuts"
https://github.com/kubevirt/kubevirt/pull/4045#discussion_r476508946

Reply:
+@EdDev 
+Quique

updating it, doesnt work out of the box, i believe that the issue is fix like #4000 presented

but soon we are going to use cloud-init, is it safe then to not do this refactor atm please ?

Reply:
I am refactoring how we run commands that's does not affect where do we take ips from, right ?

Reply:
> I am refactoring how we run commands that's does not affect where do we take ips from, right ?

True, but wonder if it will solve the flakiness you said you saw, and that i am saw here.
I will first finish the other PR i am on and then we can resync, just we wont do the same (solving flakes in this test family)

Reply:
Sorry, I do not understand what refactoring you talk about. This is a new code... login into the vmi and defining the IPv6. So this is about doing it right, not about refactoring.

I also do not understand how the usage of the wrapper helper does not work and this one does work... they are suppose to do the exact same thing, no?

Reply:
> Sorry, I do not understand what refactoring you talk about. This is a new code... login into the vmi and defining the IPv6. So this is about doing it right, not about refactoring.
> 
> I also do not understand how the usage of the wrapper helper does not work and this one does work... they are suppose to do the exact same thing, no?

It doesnt work out the box, even if "supposed" , i have assumption that adaption as PR 4000 did should solve it.
i think that even if its a new code, the current condition isnt bad, and we are soon moving to cloud-init, (very soon), so i dont see a reason to spend time on it, but first focus on the important things, such are flakiness or other tasks, but whatever you choose.
In my opinion since we know the code is going to be refacotred pretty soon (cloud init usage), i would suggest to consider the current use, as we did not once.

Reply:
found the problem, WaitUntilVMIReady is for default NS, fixing

Reply:
Done, but just this, will take care now of the other commits, dont review please yet.
About flakiness, from what i understood Quique found the problem, so i will wait for him.